src/handlers/types_planning_updates.rs

```diff
@@ -0,0 +1,208 @@
+        // this is fine under the assumption that tracking issues should only contain updates.
+        let older_than_7_days = comments
+            .last()
+            .map_or(true, |c| c.updated_at < (Utc::now() - Duration::days(7)));
+        if !older_than_7_days {
+            continue;
+        }
+        // In the future, we should reach out to specific people in charge of specific issues. For now, because our tracking
+        // method is crude and will over-estimate the issues that need updates.
+        /*
+        let mut dmed_assignee = false;
+        for assignee in issue.assignees {
+            let zulip_id_and_email = zulip_id_and_email(ctx, assignee.id.unwrap()).await?;
+            let (zulip_id, email) = match zulip_id_and_email {
+                Some(id) => id,
+                None => continue,
+            };
+            let message = format!(
+                "Type team tracking issue needs an update. [Issue #{}]({})",
+                issue.number, issue.html_url
+            );
+            let zulip_req = crate::zulip::MessageApiRequest {
+                recipient: crate::zulip::Recipient::Private {
+                    id: zulip_id,
+                    email: &email,
+                },
+                content: &message,
+            };
+            zulip_req.send(&ctx.github.raw()).await?;
+            dmed_assignee = true;
+        }
+        if !dmed_assignee {
+            let message = format!(
+                "Type team tracking issue needs an update, and was unable to reach an assignee. \
+                [Issue #{}]({})",
+                issue.number, issue.html_url
+            );
+            let zulip_req = crate::zulip::MessageApiRequest {
+                recipient: crate::zulip::Recipient::Stream {
+                    id: 144729,
+                    topic: "tracking issue updates",
+                },
+                content: &message,
+            };
+            zulip_req.send(&ctx.github.raw()).await?;
+        }
+        */
+        issues_needs_updates.push(format!("- [Issue #{}]({})", issue.number, issue.html_url));
+    }
+
+    let issue_list = issues_needs_updates.join("\n");
+
+    let message = format!("The following issues still need updates:\n\n{issue_list}");
+
+    let meeting_date_string = metadata.date_string;
+    let zulip_req = crate::zulip::MessageApiRequest {
+        recipient: crate::zulip::Recipient::Stream {
+            id: 326132,
+            topic: &format!("{meeting_date_string} planning meeting"),
+        },
+        content: &message,
+    };
+    zulip_req.send(&ctx.github.raw()).await?;
+
+    Ok(())
+}
+
+#[allow(unused)] // Needed for commented out bit above
+async fn zulip_id_and_email(
+    ctx: &super::Context,
+    github_id: i64,
+) -> anyhow::Result<Option<(u64, String)>> {
+    let bot_api_token = std::env::var("ZULIP_API_TOKEN").expect("ZULIP_API_TOKEN");
```

Comment:
Can we move this function to `zulip.rs`? That will hopefully help avoiding re-implementing it twice, and feels more consistent with the general code structure (Zulip and GitHub handling are in their own modules, handlers just call that code).

Also: I don't think this function is the right way to do this. We should use the team API and the zulip & github IDs present there, which doesn't rely on this information being available to Zulip (which AFAIK isn't guaranteed). So perhaps we can just delete it for now.

Reply:
So, I mostly copied this code from `zulip.rs` (I think, it was somewhere).

The reason I didn't just use the zulip id is because `Recipient::Private` needs to email. Not sure if this strictly required for the API, or just an artifact of the implementation here.

For now, I'll just remove this function. If I need it in the future, I'll do some related cleanups in `zulip.rs`.

Reply:
Sounds good. Yeah, not sure on email vs. Zulip user ID etc.